apps/web/__e2e__/models/editor.model.ts

```diff
@@ -287,4 +291,12 @@ export class EditorModel {
     await this.page.keyboard.press("KeyV");
     await this.page.keyboard.up("ControlOrMeta");
   }
+
+  async isUndoButtonVisible() {
+    return await this.undoButton.isVisible();
+  }
+
+  async isRedoButtonVisible() {
+    return await this.redoButton.isVisible();
+  }
```

Comment:
No need for this. Just use an `expect` assertion instead. You'll have to expose `undoButton` and `redoButton` which is totally fine.